build_travis.sh

```diff
@@ -6,14 +6,14 @@ set -e
 if [ "$GTK" = latest -o "$GTK" = "3.22.30" ]; then
 	BUNDLE="gtk-3.22.30-1"
 	if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then
-		FEATURES=gtk_3_22_30,futures-nightly
+		FEATURES=gtk_3_22_30
 	else
 		FEATURES=gtk_3_22_30,futures-stable
 	fi
 elif [ "$GTK" = "3.18" ]; then
 	BUNDLE="gtk-3.18.1-2"
 	if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then
-		FEATURES=gtk_3_18,futures-nightly
+		FEATURES=gtk_3_18
```

Comment:
futures-stable ?

Reply:
And please change in commented default in Cargo.toml

Reply:
If you remove futures-nightly from glib completely, then better remove it here too.

Reply:
I would like to keep this here for the time being as I plan to add it back based on 0.3 as soon as I figure out how to do that in glib's Cargo.toml